.gitignore

```diff
@@ -1 +1,4 @@
 node_modules
+.DS_Store
+dist/app.css
+package-lock.json
```

Comment:
This file should be checked into the repo. See: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5
```suggestion
```